components/apps_modal/interactive_dialog.tsx

```diff
@@ -0,0 +1,302 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
Why are we reusing the term interactive_dialog here (file name) - appears confusing

Reply:
This is just for the time being during the refactor to support the new field structures